staging/src/k8s.io/kubectl/pkg/cmd/patch/patch.go

```diff
@@ -146,6 +148,9 @@ func NewCmdPatch(f cmdutil.Factory, ioStreams genericiooptions.IOStreams) *cobra
 	cmd.Flags().BoolVar(&o.Local, "local", o.Local, "If true, patch will operate on the content of the file, not the server-side resource.")
 	cmdutil.AddFieldManagerFlagVar(cmd, &o.fieldManager, "kubectl-patch")
 	cmdutil.AddSubresourceFlags(cmd, &o.Subresource, "If specified, patch will operate on the subresource of the requested object.", supportedSubresources...)
+	cmdutil.AddLabelSelectorFlagVar(cmd, &o.LabelSelector)
```

Comment:
Could you please add unit tests for these 2 flags in here https://github.com/kubernetes/kubernetes/blob/00c30941260a27e6929aef84c7fdbc8f1508518c/staging/src/k8s.io/kubectl/pkg/cmd/patch/patch_test.go#L17

and integration tests preferably in https://github.com/kubernetes/kubernetes/blob/76ee3788ccbac9003e3f24de9000ebd91c27611f/test/cmd/core.sh#L423